src/main/resources/messages_de.properties

```diff
@@ -0,0 +1,52 @@
+widget.meta.title=Mit BundesIdent online ausweisen
+widget.headline-title-top=Mit BundesIdent
```

Comment:
I assume kebab-case is best practice here? Otherwise we use camelCase everywhere, that's why I was wondering...

Reply:
Kebab-case is recommended (and best practise afaik) for `.properties` and `.yaml` files in Spring (See [docs](https://docs.spring.io/spring-boot/docs/2.1.1.RELEASE/reference/html/boot-features-external-config.html) table 24.1). "Unfortunately" camelCase and snake_case are allowed as well and we also mix those up in our `application.yaml` but we mainly use kebab-case here. What do you mean with "we use camelCase" everywhere?

Reply:
I meant we use it in the code, so e.g. for variables and method names. But I see your point and let's stick with the recommendation. 👍